src/interfaces.ts

```diff
@@ -1,8 +1,11 @@
+import { GraphQLResolveInfo } from 'graphql';
+
 // Prisma Relay Cursor Connection Arguments
 export interface Options<Model, Cursor> {
   getCursor: (node: Model) => Cursor
   encodeCursor: (cursor: Cursor) => string
-  decodeCursor: (cursorString: string) => Cursor
+  decodeCursor: (cursorString: string) => Cursor,
+  info?: GraphQLResolveInfo;
```

Comment:
Nit: `info` -> `resolveInfo`

Reply:
This param is usually called `info` (see what prisma v1 used to have). Not sure if anything changed, but I'd rather not have to do `resolveInfo: info` and instead be able to do `{ info }` here. Let me know what you think